### PR TITLE
Fix get_latest_version procedure that were showing bad information

### PR DESCRIPTION
**Pull request by MasterEnoc**
- markdown: update scraper and fix get_latest_version

- rails: fix get_latest_version, it was showing an outdated
  version as the latest version

- apache: fix get_latest_version, it was showing and
  outdated version as the latest version

- cakephp: fix get_latest_version, it was showing and
  outdated version as the latest version

### CODE REVIEW

File: lib/docs/scrapers/markdown.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@ module Docs
   class Markdown < UrlScraper
     self.name = 'Markdown'
     self.type = 'simple'
-    self.base_url = 'http://daringfireball.net/projects/markdown/syntax'
+    self.release = '1.0.1'
+    self.base_url = 'https://daringfireball.net/projects/markdown/syntax'
```
Comment (simon04) [inline lib/docs/scrapers/markdown.rb:6]: I don't think the Markdown specs are versioned. AFAIK, version 1.0.1 refers to the Perl reference implementation…

**Review by simon04 — CHANGES_REQUESTED** [review]
Can you please undo the changes to markdown (see my comment)? The other changes are great.

**Comment by MasterEnoc** [comment]
should I undo all the changes of markdown? I can leave `self.release = '1.0.0'` to avoid showing it in outdated documentation

**Comment by simon04** [comment]
I'd not invent a versioning number when the specification does not provide any. Just my 2¢…

**Comment by MasterEnoc** [comment]
Fixed, please proceed to review it

**Review by simon04 — APPROVED** [review]

**Pull request merged**